lib/src/components/NxTransferListHalf/__tests__/NxTransferListHalf.test.tsx

```diff
@@ -168,6 +168,54 @@ describe('NxTransferListHalf', function() {
     expect(items[1]).toHaveTextContent('baz');
   });
 
+  it('render from 0 to 3 items', function() {
+    const view = quickRender({ items: [] });
+    const itemList = view.container.querySelector('.nx-transfer-list__item-list') as HTMLElement,
+        items = itemList.querySelectorAll('.nx-transfer-list__item');
+    expect(items.length).toBe(0);
+
+    view.rerender(
+      <NxTransferListHalf {...minimalProps} items={[{id: '1', displayName: 'foo'}]}/>
+    );
+    let group = view.getByRole('group', { name: 'foo' });
+    expect(group).toBeInTheDocument();
```

Comment:
And likewise below.  Easier to read and avoids the mutable variable.
```suggestion
    expect(view.getByRole('group', { name: 'foo' })).toBeInTheDocument();
```